tests/integration/tests/conftest.py

```diff
@@ -137,23 +193,19 @@ def instances(
 
     Bootstrap and setup networking on the first instance, if `disable_k8s_bootstrapping` marker is not set.
     """
-    if not config.SNAP:
-        pytest.fail("Set TEST_SNAP to the path where the snap is")
-
+    node_count = len(node_config)
     if node_count <= 0:
         pytest.xfail("Test requested 0 or fewer instances, skip this test.")
 
-    snap_path = (tmp_path / "k8s.snap").as_posix()
-
```

Comment:
this is gone because we can install from all kinds of things now...track, channel, revision, or file